### DISCUSSION TITLE
a way to play ryujinx ldn online?

### DISCUSSION THREAD

**Discussion by games869** [Raytwo/ARCropolis#496] [General]
so me and a friend wanted to play modded and he has a patched switch with no modchip and we both have Ryu so we tried to play using Ryu and it worked well but when we turned on mods it stopped working and I was told arc will block ldn dose anyone know how to fix that or if it can be fixed at all?

**Comment by Tac24** [comment]
if his switch is patched, there is no way to run atmosphere/CFW, that's required to run ARCpolis in the first place.

**Reply by games869** [reply]
we were trying to run the games on ryujinx the switch emulator
